cni/network/multitenancy_test.go

```diff
@@ -42,7 +46,19 @@ type getAllNetworkContainersConfigurationHandler struct {
 	err                 error
 }
 
+type cnsAPIName string
+
+const (
+	GetAllNetworkContainers cnsAPIName = "GetAllNetworkContainers"
+)
+
+var (
+	errUnsupportedAPI             = errors.New("Unsupported API")
+	errNoOrchestratorContextFound = errors.New("No CNI OrchestratorContext Found")
+)
+
 type MockCNSClient struct {
+	unsupportedAPIs                      map[cnsAPIName]bool
```

Comment:
why do we need bool? just empty struct would do, no?

Reply:
all we need to check if the key exists. We don;t really need a true / false value in the map